examples/data_sharing.tex

```diff
@@ -0,0 +1,211 @@
+\documentclass[a4paper]{article}
+\usepackage{../data}
+
+%%%%% Now we input our variable terms as glossary entries %%%%%
+
+\newglossaryentry{type}{
+	name={Company},
+	first={the ``Company''},
+	text={Company},
+	description={is a commercial undertaking}
+}
+
+\newglossaryentry{undertaking}{
+	name={King of the Eagles},
+	first={King of the Eagles, hereafter `\glsentrytext{type}'},
+	text={the \glsentrytext{type}},
+	description={are Big birds. Live on a hill}
+}
+
+\newglossaryentry{processor}{
+	name={Mirkwood Elves},
+	first={Mirkwood Elves, hereafter `the Processor'},
+	text={Data Processor},
+	description={are the Elves of the Noldor Line}
+}
+
+\newcommand{\services}{Orc-slaying}
+
+\newcommand{\companyAddress}{123 Eirie Road}
+
+
+\newcommand{\processorAddress}{Third Tree, Mirkwood}
+
+\newcommand{\processorEmail}{elves@place.mw}
+
+\title{Data Sharing Agreement between \Glsentrytext{undertaking} \& \Glsentrytext{processor}}
+
+\begin{document}
+
+\begin{titlepage}
+
+\centering
+
+\vspace{0.4\textheight}
+
+\vspace{3cm}
+
+\scshape{\Glsfmtname{undertaking}}
+
+\vspace{0.1\textheight}
+
+\scshape{and}
+
+\vspace{0.1\textheight}
+
+\scshape{\Glsfmtname{processor}}
+
+\vspace{0.2\textheight}
+
+
+\rule{0.5\textwidth}{0.4pt}
+
+Data Processing Agreement
+
+\rule{0.5\textwidth}{0.4pt}
+
+\end{titlepage}
+
+\clearpage
+
+\noindent
+This data processing agreement forms part of the contract for services, (``Principal Agreement'') between
+
+\vspace{2em}
+
+\Gls{undertaking}
+
+\vspace{2em}
+
+and
+
+\vspace{2em}
+
+\Gls{processor}
+
+\vspace{2em}
+
+\section*{WHEREAS}
+
+\begin{enumerate}
+
+	\item{\Gls{undertaking} acts as a data controller}
+	\item{\Gls{undertaking} wishes to subcontract certain Services, which imply the processing of personal data, to the \gls{processor}.}
+	\item{The Parties seek to implement a data processing agreement that complies with the requirements of the current legal framework in relation to data processing with the Regulation (EU) 2016/679 of the European Parliament and of the Council of 27 April 2016 on the protection of natural persons with regard to the processing of personal data and on the free movement of such data, and repealing Directive 95/46/EC (General Data Protection Regulation).}
+	\item{The Parties wish to lay down their rights and obligations.}
+
+\end{enumerate}
+
+IT IS AGREED AS FOLLOWS:
+
+\section{Definitions and Interpretations}
+
+\subsection{Unless otherwise defined herein, capitalized terms and expressions used in this Agreement shall have the following meaning:}
+
+\subsubsection{\textbf{``Agreement''} means this Data Processing Agreement and all Schedules;}
```

Comment:
can we add a TODO to clean up all the `\textbf`'s?

Reply:
also these really need to be a (numbered) description list. maybe add a TODO for that too

Reply:
Also perhaps change the `bf`s to emphasis. Handling this after the PR is fine though.

Reply:
This looks like another time to ask @passamo9 or a lawyer for input. Either _italic_ or **bold** text isn't a problem, once we get a clear request on what standards should be, or if there are any legal requirements here (do laws refer to 'bold text' at any point?).

Reply:
That's why handling this after the PR is probably best. We can then use and adapt emphasis to suit their needs.

Reply:
moved to #37

Reply:
let's add a TODO anyway so when we clean this up later we will remember to do that. @Andonome feel free to resolve after that's done.